### DISCUSSION TITLE
Open streams from /live with middle mouse click

### DISCUSSION THREAD

**Discussion by Nerxxy** [Chatterino/chatterino2#4356] [Ideas]
I would like to see the ability to use middle mouse to open a stream directly in the browser instead of seeing the "stream card" since when middle mouse clicking from the /live tab it opens a broken twitch tab 
![image](https://user-images.githubusercontent.com/37718547/216631876-23f6b1b5-8355-45cc-90e5-a3b5e96ccb7d.png)

**Comment by pajlada** [comment]
This is more of a bug in `/live` where it doesn't properly give the channel name to this feature

**Reply by Nerxxy** [reply]
Although it is a bug i would still personally prefer to be taken to the stream, just makes things more seamless imo

**Reply by pajlada** [reply]
yeah you're right, for `/live` this functionality doesn't make sense. It would've opened `/popout/forsen/viewercard/forsen` which is kind of useless

**Reply by Nerxxy** [reply]
plus if u wanted to see the card in the first place u can just left click the name and chatterinos native one will show
